js/stdlib/Dockerfile

```diff
@@ -19,6 +19,7 @@ COPY package.json package-lock.json /stdlib/
 COPY ts/* /stdlib/ts/
 COPY *.mjs /stdlib/
 COPY types/* /stdlib/types/
+COPY test/* /stdlib/test/
```

Comment:
Similarly, these tests wouldn't be included in the actual package, but would just be our thing